services/galley/src/Galley/API/MLS/Util.hs

```diff
@@ -50,10 +77,29 @@ getLocalConvForUser ::
   Local ConvId ->
   Sem r Data.Conversation
 getLocalConvForUser qusr lcnv = do
-  conv <- getConversation (tUnqualified lcnv) >>= noteS @'ConvNotFound
+  gtc <- getGlobalTeamConversationById lcnv
```

Comment:
This is a bit strange. Can't we simply accept that a GTC does not have a creator? All this fudging around with the creator should not be necessary.

Reply:
This has large ramifications… I think it makes sense to clean it up when we do the conversation sum type.